bin/adhoc-scripts/parseUnifiedCampaigns.py

```diff
@@ -276,14 +311,24 @@ def main():
             key = proxy
         hdict = {'cert': cert, 'key': key, 'pycurl': True}
         mgr = ReqMgrAux(opts.url, hdict, logger=logger)
-    if opts.dburi:
+    if opts.cin:
```

Comment:
I suggest to rely on another option, e.g. add-test-campaigns here and re-use opts.fin as input file option. So this code can be replaced as
```
if opts.test_campaigns:
```